docs/Makefile

```diff
@@ -61,147 +62,176 @@ help:
 	@echo "  coverage   to run coverage check of the documentation (if enabled)"
 
 clean:
-	rm -rf $(BUILDDIR)/*
+	rm -rf $(BUILDDIR)
+	rm -rf $(STAGINGDIR)
+	# Remove folders that have since been relocated into
+	# $(STAGINGDIR).  This allows `task_sphinx_precheck.sh` to
+	# run, even if a commit that predates $(STAGINGDIR) was
+	# previously run on that node.
 	rm -rf gen_modules
```

Comment:
Right now, these are intentionally present, so that CI nodes that have the `docs/tutorials` folders get them cleared out over time, and so that PRs based on branches before the `$(STAGINGDIR)` don't cause the issue to reappear.  After the change has been propagated, then I want to make a new PR to remove these lines.

Reply:
Added a `TODO(Lunderberg)` to remind me to go back and change them later.